types/src/front/assistant/agent.ts

```diff
@@ -100,23 +100,21 @@ export type AgentUserListStatus = "in-list" | "not-in-list";
 /**
  * Defines strategies for fetching agent configurations based on various
  * 'views':
- * - 'list': Retrieves all agents within the user's list, including their
- *   private agents, agents from the workspace and global scope, plus any
- *   published agents they've added to their list (refer to
- *   AgentUserRelationTable).
+ * - 'list': Retrieves all active agents accessible to the user
```

Comment:
**Note on `AgentsGetViewType`**

While doing the PR, realized:
- conversationId view is broken at the moment (not by this PR);
- list & assistant-search are becoming redundant
- we could alomost say that conversationId is also redundant, but it allows sharing convos with personal agents so we can keep it for this purpose

Will do an immediate follow-up PR "Clean up agent views" fixing 1 and merging assistant-search+list (a few intricacies, don't want to overload this PR)

Reply:
yep true, that was the intricacies :) (and why I said "are becoming" rather than "are")
will take care of that too